src/EditorFeatures/DiagnosticsTestUtilities/CodeActions/CSharpCodeFixVerifier`2+Test.cs

```diff
@@ -54,7 +65,7 @@ public Test()
                         }
                     }
 
-#if !CODE_STYLE
+#if !CODE_STYLE // TODO: Add support for Options based tests in CodeStyle layer
```

Comment:
`remaningOptions` are non-editorconfig based options correct? Will they ever be supported in CodeStyle layer?

Reply:
This was an unintended outcome of merge conflict resolution. The comment change has been removed.